website/screens/overview/introduction/IntroductionPage.tsx

```diff
@@ -129,20 +129,50 @@ const sections = [
             These are the pre-defined design opinions explained by writing. They
             are contained on this website and describe, with a high level of
             detail, aspects like usage, existing variants, or design
-            specifications of the identified patterns.{" "}
+            specifications of the identified patterns.
           </DxcParagraph>
         ),
       },
       {
-        title: "Adobe XD Library",
+        title: "UXPin",
         content: (
           <DxcParagraph>
-            Implementing the previously described Design Guidelines so that
-            product designers can easily drag and drop the pre-created
-            components into their designs, obtaining automatic updates of these
-            components as the Design System evolves.
+            In the past, we used to rely on Adobe XD for our Design UI Kit. But
+            we recently decided to migrate all the assets to{" "}
+            <DxcLink href="https://www.uxpin.com/">UXPin</DxcLink> to start a
+            new and exciting journey. This decision brings, in addition to the
+            well-known image-based classic design library, a new feature, the
+            prototyping library.
           </DxcParagraph>
         ),
+        subSections: [
+          {
+            title: "Classic Design Library",
+            content: (
+              <DxcParagraph>
+                Implementing the previously described Design Guidelines so that
+                product designers can easily drag and drop the pre-created
+                components into their designs, obtaining automatic updates of
+                these components as the Design System evolves.
+              </DxcParagraph>
+            ),
+          },
+          {
+            title: "Prototyping Library",
+            content: (
+              <DxcParagraph>
+                The prototyping library is a Halstack CDK component-based design
+                library for fast prototyping, reducing the gap between designers
+                and developers. Also,{" "}
+                <DxcLink href="https://www.uxpin.com/merge">
+                  UXPin Merge
```

Comment:
I would add `newWindow` to UXPin Merge documentation link.

Reply:
Totally agree.